app/models/activity.rb

```diff
@@ -64,4 +64,8 @@ def display_name
   def points
     observations.sum(:points)
   end
+
+  def recorded_on
```

Comment:
To me "recorded on" would be the date the activity was recored, so `created_at` not `happened_on`.

Reply:
This method is not used now so have removed it